app/services/data/get-workload-report-views.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+const CapacityType = require('../../constants/capacity-type')
+
+// TODO: Expand with id
+module.exports = function (type) {
+  if (type === undefined ||
+      (CapacityType.REGION !== type &&
+      CapacityType.TEAM !== type &&
+      CapacityType.LDU !== type)) {
+    throw new Error(type + ' should be a location of REGION, TEAM or LDU')
+  }
+  var sql = 'SELECT * from app.' + type
```

Comment:
This should be rewritten in `knex` lingo i.e. `return knex(type)` should be enough to do a select all on the passed in table.